src/views/vmd-rdv.view.ts

```diff
@@ -243,21 +246,25 @@ export abstract class AbstractVmdRdvView extends LitElement {
                         <h2 class="row align-items-center justify-content-center mb-5 h5 px-3">
                             <i class="bi vmdicon-calendar2-check-fill text-success me-2 fs-3 col-auto"></i>
                             <span class="col col-sm-auto">
-                                ${lieuxDisponibles.length} Lieu${Strings.plural(lieuxDisponibles.length, 'x')} de vaccination avec des disponibilités
+                                ${lieuxDisponibles.length} Lieu${Strings.plural(lieuxDisponibles.length, 'x')} de vaccination avec des ${this.searchType==='chronodose' ? 'chronodoses' : 'disponibilités'}
                             </span>
                         </h2>
                     ` : html`
                         <h2 class="row align-items-center justify-content-center mb-5 h5">
                           <i class="bi vmdicon-calendar-x-fill text-black-50 me-2 fs-3 col-auto"></i>
                           <span class="col col-sm-auto">
-                            Aucun créneau de vaccination trouvé
+                            Aucun créneau ${this.searchType==='chronodose' ? 'chronodose' : 'de vaccination'} trouvé
                           </span>
                         </h2>
                         <div class="px-3 mb-5">
                           <em>Nous n’avons pas trouvé de <strong>rendez-vous de vaccination</strong> Covid-19
                             sur les plateformes de réservation. Nous vous recommandons toutefois de vérifier manuellement
                             les rendez-vous de vaccination auprès des sites qui gèrent la réservation de créneau de vaccination.
                             Pour ce faire, cliquez sur le bouton “vérifier le centre de vaccination”.</em>
+                            ${this.searchType === 'chronodose' ? html`
+                                <em>Si vous êtes déjà éligible, vous pouvez <a class="text-decoration-underline" href="" @click="${() => this.updateSearchTypeTo('standard')}">consulter les créneaux classiques</a>.</em>
+                            `:``}
+                            
```

Comment:
La balise `<em>` est plutôt à utiliser pour mettre en avant quelques mots ([source](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em#usage_notes)).

Pour mettre le texte en italique, il faut mieux utiliser la classe bootstrap [`fst-italic`](https://getbootstrap.com/docs/5.0/utilities/text/#font-weight-and-italics).

Est-ce que tu peux faire le changement ?